deployments/deployment.yaml

```diff
@@ -118,6 +118,11 @@ objects:
                   - introspect
                   - https://cdn.redhat.com/content/dist/layered/rhel8/x86_64/ansible/2/os
             image: ${IMAGE}:${IMAGE_TAG}
+            command:
+              - /content-sources
+              - api
+              - consumer
+              - instrumentation
```

Comment:
maybe this was a temporary change, but i don't think we want consumer to run as part of this right?

Reply:
thanks! I changed locally and I have committed at some point!